cleo_plugins/FileSystemOperations/FileSystemOperations.cpp

```diff
@@ -105,7 +105,8 @@ class FileSystemOperations
         }
         else
         {
-            path = OPCODE_READ_PARAM_STRING();
+            OPCODE_READ_PARAM_STRING(str);
+            path = str;
```

Comment:
you can remove `else` because then ends with `return`.

```
OPCODE_READ_PARAM_STRING(str);
CLEO_SetScriptWorkDir(thread, str);
return OR_CONTINUE;
```

Reply:
Fixed